init.lua

```diff
@@ -229,7 +229,9 @@ end)
 function CheckOptions(data, entity, distance)
 	if distance and data.distance and distance > data.distance then return false end
 	if data.job and not JobCheck(data.job) then return false end
+	if data.excludejob and JobCheck(data.job) then return false end
```

Comment:
Shouldn't the JobCheck look at the `data.excludejob` and not `data.job` due to then it would be checking the job in properties so the code should look like 
`if data.excludejob and JobCheck(data.excludejob) then return false end`